### DISCUSSION TITLE
socks5/http代理支持，字体字号设置，输入窗口位置过于偏上，及多行文本输入问题。

### DISCUSSION THREAD

**Discussion by hongyi-zhao** [nextai-translator/nextai-translator#518] [General]
1. 考虑到 Open AI 默认支持的国家问题，是否可以添加 socks5/http 代理支持？
2. 字体字号设置可否自定义？
3. 输入窗口位置过于偏上问题，如下所示：
![image](https://user-images.githubusercontent.com/11155854/228400177-1211b4cc-3d90-4203-9549-424fa0e5bbc0.png)

可见，第一行输入的字被覆盖了一半的高度。
4. 当前，无法输入多行文本。`Enter` 会自动触发执行动作。我的目标如下：

允许用户输入多行文本，并将触发执行动作绑定到其他的快捷键，比如 `Escape Enter`。

**Comment by maltoze** [comment]
换行用 <kbd> Shift </kbd> + <kbd> Enter </kbd>

**Comment by maltoze** [comment]
#523 会修复样式的问题

**Reply by luxueqiang** [reply]
字体大小调节功能能否支持呢？大屏幕看不清翻译文字
